cmd/solidfire-exporter/main.go

```diff
@@ -36,12 +36,26 @@ func init() {
 	} else {
 		viper.BindEnv(solidfire.ListenPortFlag, solidfire.ListenPortFlagEnv)
 	}
-
-	viper.BindEnv(solidfire.UsernameFlag, solidfire.UsernameFlagEnv)
```

Comment:
If we remove these here, can we still pass username/password through an environment variable?
As I recall, we want to support username and password via an Environment variable or config file, but want to drop support for having them as command-line flags.